docs/submission/submitted-data.md

```diff
@@ -8,107 +8,90 @@ Program members can track the progress of their program’s clinical and molecul
 
 When a program member logs in to the ARGO Data Platform, and navigates to the **Program Services** area in the top menu, they will land on the program dashboard. If a member belongs to multiple programs, each program has its own dashboard and submission sections.
 
-Program progress is summarized within the following dashboard "cards":
-
-## Program Statistics
-
-This card aggregates the following program statistics:
-![Dashbard Program Statistics](/assets/submission/dashboard-stats.png)
-
-#### Registered Donors
-
-Once samples are registered, each corresponding donor will be counted as a percentage out of the number of committed program donors.
-
-#### % Donors with all Core Clinical Data
-
-Once the minimum amount of required clinical data has been submitted for a donor, they are considered **"clinically complete"**.
-
-> **A donor must be clinically complete before any of their molecular analysis files are released to the program members.**
-
-Check out [Clinical Data Validation Rules](/docs/submission/clinical-data-validation-rules) for more information on how clinical completion is calculated.
-
-#### % Donors with Tumour & Normal
 
-_Coming Soon:_ Once a donor has both a tumour and matched normal raw sequencing data [submitted through Song](/docs/submission/submitting-molecular-data), they will be counted towards this percentage.
 
-#### Donors in Molecular Data Processing
+## Program Overview: Statistics and Progress
 
-_Coming Soon:_ This is the number of donors that currently have molecular data being processed in the [analysis pipeline](/docs/analysis-workflows/analysis-overview).
+The first half of the Program Dashboard highlights import metrics of progress and statistics to display a programs completion status of molecular and clinical data against the number of donors that have been committed for submission to ARGO. 
```

Comment:
minor update: `display a program's completion` (missing `'s` at end of `programs)